apps/right-to-rent-check/index.js

```diff
@@ -137,6 +139,9 @@ module.exports = {
     '/confirm': {
       next: '/complete'
     },
+    '/exit-dont-need-check': {
```

Comment:
@karypun are working on urls, this was just a placeholder but this sounds better